terraform/demo/main.tf

```diff
@@ -127,6 +127,22 @@ module "data_share_service" {
   ]
 }
 
+module "len" {
+  source                      = "../modules/len"
+  environment                 = local.env
+  region                      = data.aws_region.current.name
+  schedule                    = "rate(1 minute)"
```

Comment:
do we want this frequency for test as well, or do they want it more realistic?

Reply:
Going to start with this and tweak